docker-compose.yml

```diff
@@ -11,6 +11,9 @@ services:
       - "8081:8080"
       - "5016:5006"
   pensjon-brevbaker:
+    depends_on:
```

Comment:
Er det nødvendig at brevbaker er avhengig av pdf-bygger? Nå vil vel brevbaker containeren vente med å starte?

Reply:
Jeg lurer litt på om vi ønsker noe særlig rigiditet i dependencies i docker-compose for lokal kjøring?

Reply:
Fjerner noen dependencies. Startet med skribenten-db siden db må starte og være oppe før skribenten, men ellers er det egentlig ikke nødvendig nei.